x-pack/filebeat/module/mssql/log/config/config.yml

```diff
@@ -0,0 +1,15 @@
+type: log
+paths:
+  {{ range $i, $path := .paths }}
+- {{$path}}
+  {{ end }}
+exclude_files: [".gz$"]
+
+multiline.pattern: '^\d\d'
+multiline.negate: true
+multiline.match: after
+
+{{ if .convert_timezone }}
```

Comment:
@sayden @jsoriano @exekias I'm thinking we should remove this config option completely and have `add_locale` always enabled and no option to disable it.  It also means the `convert_timezone ` config option completely disappears. WDYT?

NOTE: This only applies to the new modules so here it would not be a breaking change if we follow up with this.

Reply:
+1 to remove `convert_timezone`. I think the only reason to have it would be to support ES versions that don't have the `timezone` field in the date processor, and we are not supporting them in any case.

> NOTE: This only applies to the new modules so here it would not be a breaking change if we follow up with this.

I would consider changing it even on existing modules, I'd say it is more a bugfix/enhancement. It would only be breaking for cases where the logs are being parsed with an incorrect timezone, and in these cases the change would be for good.

For cases where this needs to be customized or this is not wanted for any reason there is still the workaround of using processors.

Reply:
+1

Reply:
@sayden Could you open a PR for MSSQL to remove it?

@jsoriano Could you open a PR for a module with your proposed change. I think I need to see the code change to figure out if it's really not breaking :-)

Reply:
PR open to follow with this discussion: https://github.com/elastic/beats/pull/12410